### PR TITLE
Fix some issues of  #3512 and add README.md to librispeech/ssl1 recipe.

### PR DESCRIPTION
**Pull request by Jzmo**
This PR includes following content:
1.  Remove `--use_lm` and other language model related parameters in egs2/TEMPLATE/hubert.sh
2.  Remove `_org` subfolder in stage 3 of egs2/TEMPLATE/hubert.sh to fix the "failed to find data folder" issue.
3.  Add symbolic of Librispeech data preparation in egs2/librispeech/ssl1/local/
4.  Reset bias initialization of `torch.nn.GroupNorm` in `espnet2/torch_utils/initialize.pyL83`
5.  Add README.md to librispeech/ssl1

### CODE REVIEW

**Comment by kan-bayashi** [comment]
I can merge soon if you fix the CI.

File: egs2/librispeech/ssl1/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Stage 6 and 7 collect stats of train/valid set and train the hubert model respectively. These two stages have the same functionality as stage 10 and 11 of asr recipes. The only difference is we call `espnet2.bin.hubert_train` instead of `espnet2.bin.train` for training.
+
+    ./run.sh --stage 6 --stop-stage 7
+	
+Note that stage 5-7 could run for multiple times, which specified by `pretrain_start_iter` and `pretrain_start_iter`. You may find the default value in run.sh:
+
+    pretrain_start_iter=0
+    pretrain_stop_iter=2
+	
+That refer to 3 iterations, n_clusters_iter[0-2] and feature_iter[0-2] specify the number of clusters and the feature type used for k-means clustering of different iterations. Follow the [Hubert](https://arxiv.org/pdf/2106.07447.pdf) settings of  base model, we use mfcc and 100 cluster for the iteration 0, and extract the latent features from transformer layer 6 of HuBERT model pre-trained in prtrains iteration(HuBERT6), 500 clusters for iteration 1, and (HuBERT9) 500 clusters for iteration 2. Each iterations have different config files. Please refer to conf/tuning/train_asr_hubert_base_960h_pretrain_it*.yaml
```
Comment (sw005320) [inline egs2/librispeech/ssl1/README.md:31]:
```suggestion
That refers to 3 iterations, n_clusters_iter[0-2] and feature_iter[0-2] specify the number of clusters and the feature type used for k-means clustering of different iterations. Follow the [Hubert](https://arxiv.org/pdf/2106.07447.pdf) settings of the base model, we use mfcc and 100 clusters for the iteration 0, and extract the latent features from transformer layer 6 of HuBERT model pre-trained in prtrains iteration(HuBERT6), 500 clusters for iteration 1, and (HuBERT9) 500 clusters for iteration 2. Each iteration has a different config file. Please refer to conf/tuning/train_asr_hubert_base_960h_pretrain_it*.yaml
```

File: egs2/librispeech/ssl1/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    the percentage you want to used to train the k-means model.
+These parameters can be settled in run.sh and pass to hubert.sh
+
+    ./run.sh --stage 5 --stop-stage 5
+	
+Stage 6 and 7 collect stats of train/valid set and train the hubert model respectively. These two stages have the same functionality as stage 10 and 11 of asr recipes. The only difference is we call `espnet2.bin.hubert_train` instead of `espnet2.bin.train` for training.
+
+    ./run.sh --stage 6 --stop-stage 7
+	
+Note that stage 5-7 could run for multiple times, which specified by `pretrain_start_iter` and `pretrain_start_iter`. You may find the default value in run.sh:
```
Comment (sw005320) [inline egs2/librispeech/ssl1/README.md:26]:
```suggestion
Note that stage 5-7 could run multiple times, which is specified by `pretrain_start_iter` and `pretrain_start_iter`. You may find the default value in run.sh:
```

File: egs2/librispeech/ssl1/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+## INTRODUCTION
```
Comment (sw005320) [inline egs2/librispeech/ssl1/README.md:1]: Can you throw a grammar check (e.g., Grammarly)?

**Comment by sw005320** [comment]
After this PR is merged, can you also add this new feature in our main README (https://github.com/espnet/espnet/blob/master/README.md) to advertise this?

**Comment by Jzmo** [comment]
> After this PR is merged, can you also add this new feature in our main README (https://github.com/espnet/espnet/blob/master/README.md) to advertise this?

Okay, I will prepare it.

**Comment by sw005320** [comment]
Thanks, @Jzmo!!

**Pull request merged**